src/query_plan/operation.rs

```diff
@@ -1043,6 +1182,13 @@ impl NormalizedFragmentSpreadSelection {
         })
     }
 
+    pub(crate) fn with_updated_selection_set(
+        &self,
+        _selection_set: Option<NormalizedSelectionSet>,
+    ) -> Self {
+        unimplemented!("unsupported")
```

Comment:
```suggestion
        todo!("unsupported")
```
since we use `todo!()` everywhere else, can we do so here so there's only one thing to grep for? or is it intentionally different?